internal/controller/data_plane_policies_workflow.go

```diff
@@ -119,3 +127,24 @@ func gatewayComponentsToSync(gateway *machinery.Gateway, componentGroupKind sche
 	}
 	return nil
 }
+
+func getGatewayControllerNames(envName string, defaultGatewayControllerName string) []gatewayapiv1.GatewayController {
+	envValue := env.GetString(envName, defaultGatewayControllerName)
+	gatewayControllers := utils.Map(strings.Split(envValue, ","), func(c string) gatewayapiv1.GatewayController {
+		return gatewayapiv1.GatewayController(strings.TrimSpace(c))
+	})
+
+	if envValue == defaultGatewayControllerName {
+		return gatewayControllers
+	}
+	return append(gatewayControllers, gatewayapiv1.GatewayController(defaultGatewayControllerName))
+}
+
+func defaultGatewayControllerName(controllerName gatewayapiv1.GatewayController) gatewayapiv1.GatewayController {
+	if slices.Contains(istioGatewayControllerNames, controllerName) {
```

Comment:
Can we please use `github.com/samber/lo` here and avoid importing another package (even if stdlib)?